cni/nnsgrpcclient/nnsgrpclient.go

```diff
@@ -0,0 +1,89 @@
+// Node network service grpc client
+package nnsgrpcclient
+
+import (
+	"context"
+	"fmt"
+	"github.com/Azure/azure-container-networking/log"
+	nns "github.com/Azure/azure-container-networking/proto/nodenetworkservice/3.302.0.744"
+	"google.golang.org/grpc"
+	"time"
+)
+
+const (
+	nnsPort = "6668"
```

Comment:
how set in stone is this? should they ever change, code would need to as well. should this be config instead?

Reply:
My understanding is that once it is set, these things don't really change since it is a host level component and will require deployment across the fleet for any client depending on it. Having it in the config is definitely better, but regardless we will need to produce a new package. I'll ask node service team if it can be changed in future.